specs/serving/knative-api-specification-1.0.md

```diff
@@ -1040,7 +1040,7 @@ resource.
 <br>
 Min: 1
 <br>
-Max: 1
+Max: MAY be larger than 1
```

Comment:
I'd almost prefer to just remove the entire line. But I'm ok with this if people want to keep it.

Reply:
Removing the entire line LGTM as well. Let's see if other people have opinions.

Reply:
May want to indicate that it's acceptable to have Max be 1

Reply:
Or that implementations may implement a max >= min